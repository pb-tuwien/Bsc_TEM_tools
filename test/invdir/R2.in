My beautiful survey

1	3	3	0	1

1  << scale for triangular mesh

0 << num_regions
res0.dat

1	0	<< inverse_type, target_decrease

1	0	<< data type (0=normal;1=log), regularization type

1	10	2	1	<< tolerance, max_iterations, error_mod, alpha_aniso

0.01	0.02	-10000000000.0	10000000000.0	<<  a_wgt, b_wgt, rho_min, rho_max

5	<< num_poly
640087.3225456941	162.067
640109.0255546428	162.067
640109.0255546428	155.6937164383748
640087.3225456941	155.6937164383748
640087.3225456941	162.067

72	<< num_electrodes
1	73
2	72
3	71
4	70
5	69
6	68
7	67
8	66
9	65
10	64
11	63
12	62
13	61
14	60
15	59
16	58
17	57
18	56
19	55
20	54
21	53
22	52
23	51
24	50
25	49
26	48
27	47
28	46
29	45
30	44
31	43
32	42
33	41
34	40
35	39
36	38
37	37
38	36
39	35
40	34
41	33
42	32
43	31
44	30
45	29
46	28
47	27
48	26
49	25
50	24
51	23
52	22
53	21
54	20
55	19
56	18
57	17
58	16
59	15
60	14
61	13
62	12
63	11
64	10
65	9
66	8
67	7
68	6
69	5
70	4
71	3
72	2

