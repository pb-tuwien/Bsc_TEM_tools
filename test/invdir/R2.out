

 >> R  2    R e s i s t i v i t y   I n v e r s i o n   v4.10 <<

 >> D a t e : 15 - 12 - 2024
 >> My beautiful survey                                                             
 >> I n v e r s e   S o l u t i o n   S e l e c t e d <<
 >> Reading start resistivity from res0.dat                      
 >> R e g u l a r i s e d   T y p e <<
 >>   L i n e a r    F i l t e r    <<
 >> L o g - D a t a   I n v e r s i o n <<
 >> N o r m a l   R e g u l a r i s a t i o n <<
 >> D a t a   w e i g h t s   w i l l   b e  m o d i f i e d <<


 Processing dataset   1


 Measurements read:  2016     Measurements rejected:     0
   Geometric mean of apparent resistivities:  0.12455E+02

 >> Total Memory required is:          1.052 Gb

   Iteration   1
     Initial RMS Misfit:        39.92       Number of data ignored:     0
     Alpha:          61.077   RMS Misfit:        0.54  Roughness:        9.648
     Step length set to   1.00000
     Final RMS Misfit:        0.54

 Solution converged - Outputing results to file

 Calculating sensitivity map


 Processing dataset   2


 End of data:  Terminating


                     Timing Summary
                     ==============
     Total time taken:                               300. s
     Percent time taken forming element equations:    0.3 %
     Percent time taken solving forward  equations:  31.5 %
     Percent time taken forming Jacobian:             1.2 %
     Percent time taken solving inverse equations:   39.4 %
     Percent time taken calculating res. matrix:      0.0 %

                     Timing Details
                     ==============
     Number of calls to form element equations:        60
     Average time taken in call:                      0.0 s
     Number of calls to solve forward equations:     4320
     Average time taken in call:                      0.0 s
     Number of calls to form Jacobian:                  1
     Average time taken in call:                      3.5 s
     Number of calls to solve inverse equations:        1
     Average time taken in call:                    118.3 s
     Number of calls to calculate res. matrix:          1
     Average time taken in call:                      0.1 s
